### DISCUSSION TITLE
Update stepca

### DISCUSSION THREAD

**Discussion by JannikZorn** [smallstep/certificates#1592] [Q&A] [answered]
Hello, 
I'm using Smallstep CA/0.24.2 (linux/amd64) on ubuntu. I want to update to version Smallstep CA/0.25.0 (linux/amd64) without losing my certs and configs. I didn't find any documentation or thread about updating the step ca. 
Is there any "easy" way to update the existing step ca or do I have to install the new version of step ca and copy all certs and config files?

Thanks in advance!

**Comment by tashian** [comment, marked as answer]
To upgrade, you can just follow the install instructions again, and then restart step-ca. Your existing certs and config will not be touched.
